lib/core/app/models/concerns/association_resource/has_many_resources.rb

```diff
@@ -20,7 +20,7 @@ def query_resource(model)
       yield query if block_given?
       query = query.where(type_column => model.class.resource_name) if type_column.present?
 
-      query.find
+      query.all
```

Comment:
I think this will break our stubbed resource spec

Reply:
Ok, I've tried on my local (current whistler services master and this branch for ROS) and all seems to have passed! I'll merge this